styles.css

```diff
@@ -122,6 +143,92 @@ header a:hover {
   font-weight: lighter;
 }
 
+.project-panel img {
+  width: 100%;
+}
+
+.project-panel p {
+  font-size: 0.8rem;
+}
+
+#proj1::before {
```

Comment:
the ::before pseudo elements all have the same css on them except for the content. can you apply that css using a class then just set the content using the IDs, to reduce code duplication

Reply:
good idea